frontend/src/components/Home.js

```diff
@@ -128,6 +129,26 @@ class Home extends React.Component {
         }
       });
     });
+
+    this.map.on("click", e => {
+      stateLayers.forEach(element => {
+        const tracts = this.map.queryRenderedFeatures(e.point, {
+          // TODO: get all layers using a .map on stateLayers instead of hardcoding IL
+          layers: ["mapbox://meghabyte.ac7v02uw"]
+        });
+
+        if (tracts.length > 0) {
+          this.setState({
+            ready: true,
```

Comment:
is `ready` used to represent that a tract was clicked? if so, could we use a more descriptive var to mirror `tractSelected` like `tractClicked` or something like that?